tiatoolbox/__init__.py

```diff
@@ -88,8 +91,10 @@ def read_registry_files(path_to_registry: str | Path) -> dict | str:
 
 
     """
+    if isinstance(path_to_registry, Path):
```

Comment:
```suggestion
```

You do not need an if statement here. "path_to_registry = str(path_to_registry)" should be enough.

Reply:
Yes `Path` here causes mypy to fail. It seems that you cannot use `/` on `Path` objects, `/` only works for `PurePath` objects.